["Error 3217: callstmt.vdm at 27:10 Unexpected type for argument 1","Error 3213: callstmt.vdm at 31:5 Operation DummyCall is not in scope","Error 3216: callstmt.vdm at 35:5 Expecting 0 arguments"]